data/processed/port_wsc.html

```diff
@@ -3984,10 +3985,10 @@
         </li>
         <p></p>
         <li>
-            The user changed his password from "GrWQWu8JyC" toÂ&nbsp;"willow-towered Canopy
-            Huntertropic wrestles" as
-            <b> it </b> was easy to remember.
-            <p></p><em> Trecho: </em><b> it </b> was easy to remember.
+            O usuário mudou sua senha "GrWQWu8JyC" para "willow-towered Canopy
+            Huntertropic wrestles" porque
+            <b> ela </b> era fácil de lembrar.
+            <p></p><em> Trecho: </em><b> ela </b> era fácil de lembrar.
```

Comment:
será que não vale a pena traduzir para o português também a senha? (até pq em inglês, há o risco de dar "out of vocabulary" em relaćão ao vocabulário que nosso modelo conhece)

Reply:
O duro é que essa frase é muito bizarra. Não como traduzi-la. :/